spec/rubocop/cop/style/if_unless_modifier_of_if_unless_spec.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::IfUnlessModifierOfIfUnless do
+  include StatementModifierHelper
+
+  subject(:cop) { described_class.new(config) }
+  let(:config) do
+    RuboCop::Config.new({})
+  end
+
+  it 'provides a good error message' do
+    source = 'condition ? then_part : else_part unless external_condition'
+    inspect_source(cop, source)
+    message = cop.offenses.first.message
+    expect(message).to include('Avoid')
+    expect(message).to include('unless')
+    expect(message).to include('modifier')
+  end
+
+  context 'ternary with modifier' do
+    let(:source) do
+      'condition ? then_part : else_part unless external_condition'
+    end
+
+    it 'registers an offense' do
+      inspect_source(cop, source)
+      expect(cop.offenses.size).to eq(1)
+    end
+
+    xit 'does auto-correction' do
```

Comment:
I don't see auto-correction in the cop, so I guess you can delete this instead of disabling it.